brigade/static/scss/molecules/_card.scss

```diff
@@ -5,32 +5,119 @@
 //
 
 .card {
+  font-weight: inherit;
+
+  display: flex;
+  flex-direction: row;
+
+  margin: 15px 0;
+
   box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
+  flex-wrap: wrap;
+
+  /* Make the parts of each card full-width by default */
+  & > * {
+    flex: 100% 1;
+  }
+
+  &:hover {
+    box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
+  }
+
+  /* Make sure the first element in each part of the card doesn't have a top margin */
+  & > * > :first-child {
+    margin-top: 0;
+  }
+
+  /* Make sure the last element in each part of the card doesn't have a bottom margin */
+  & > * > :last-child {
+    margin-bottom: 0;
+  }
 }
 
-.card:hover {
-  box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
+a.card {
+  color: inherit;
+  /* Remove any hover border if the card is a link */
+  &:hover {
```

Comment:
For the future, maybe we should have some `:active` styling too (for accessibility sake)